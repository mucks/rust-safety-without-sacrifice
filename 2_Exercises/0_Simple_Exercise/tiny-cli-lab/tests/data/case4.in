# Longer line and punctuation
UPPER hello, rust class!
REV one two three four
ADD 12345 67890
MUL 9 9
COUNT abracadabra
