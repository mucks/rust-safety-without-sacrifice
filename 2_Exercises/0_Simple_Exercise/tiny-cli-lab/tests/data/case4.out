HELLO, RUST CLASS!
four three two one
80235
81
5
